content/en/platform/corda/5.0-alpha/deploying/deployment-tutorials/tutorial-k8s.md

```diff
@@ -1,227 +1,282 @@
+## Download and Register Docker Images
 
-* [Docker Desktop](#install-and-configure-docker-desktop)
-* [minikube](#install-minikube)
+The Corda Docker images must be in a Docker registry that is accessible from the Kubernetes cluster in which Corda will run. The images are provided in a `tar` file which can be loaded into a local Docker engine and then pushed from there to the registry.
 
-Docker Desktop provides a simpler user experience but commercial use in larger enterprises requires a paid subscription.
-See [Do I need to pay to use Docker Desktop?](https://docs.docker.com/desktop/faqs/general/#do-i-need-to-pay-to-use-docker-desktop) for more details.
+1. Download [corda-worker-images.tar]()**add link.
 
-### Install and Configure Docker Desktop
-
-1. Install Docker Desktop.
-   * [macOS](https://docs.docker.com/desktop/mac/install/)
-   * [Windows](https://docs.docker.com/desktop/windows/install/)
-   * [Linux](https://docs.docker.com/desktop/install/linux-install/)
-2. [Enable Kubernetes](https://docs.docker.com/desktop/kubernetes/#enable-kubernetes) in Preferences.
-3. Configure your Kubernetes cluster with at least 6 CPU and 8 GB RAM.
-   * For macOS, configure the [resources](https://docs.docker.com/desktop/settings/mac/#resources) in the Docker Desktop Preferences.
-   * For Linux, configure the [resources](https://docs.docker.com/desktop/settings/linux/#resources) in the Docker Desktop Preferences.
-   * For Windows, configure the WSL settings in the [.wslconfig](https://docs.microsoft.com/en-us/windows/wsl/wsl-config#configuration-setting-for-wslconfig) file.
-
-### Install minikube
-
-1. Install [minikube](https://minikube.sigs.k8s.io/docs/start/).
-2. Start minikube with at least 8 GB memory and 6 CPUs:
-
-    ```sh
-    minikube start --memory 8000 --cpus 6
-    ```
-
-3. If you don't already have the `kubectl` CLI installed, set up the following alias:
-
-    ```sh
-    alias kubectl="minikube kubectl --"
-    ```
-
-## Setup for Helm Installation
-
-The Kubernetes configuration for Corda is packaged in a Helm chart.
-The chart is installed into a Kubernetes namespace using the Helm CLI.
+2. Inflate and load the `corda-worker-images.tar` file into the local Docker engine with the following command:
+   ```shell
+   docker load -i corda-worker-images.tar
+   ```
 
-### Install Helm
+3. Retag each image using the name of the registry to be used and push the image. The following is an example script to automate this. It takes the target Docker registry as an argument.
+   ```shell
+   #!/bin/bash
+   if [ -z "$1" ]; then
+    echo "Specify target registry"
+    exit 1
+   fi
+
+   declare -a images=(
+    "corda-os-rpc-worker" "corda-os-flow-worker"
+    "corda-os-member-worker" "corda-os-p2p-gateway-worker"
+    "corda-os-p2p-link-manager-worker" "corda-os-db-worker"
+    "corda-os-crypto-worker" "corda-os-plugins" )
+   tag=5.0.0.0-BetaProgram-alpha.1-RC01
```

Comment:
this tag needs to reflect whatever we expect the final version to be

@ronanbrowne - is this going to be `5.0.0.0-BetaProgram-alpha.1`?

Reply:
@davidcurrie apologies just spotted this now.

yes so in this case it will be `5.0.0.0-Eagle` as per yesterday product decision on code names